nixos/modules/hardware/new-lg4ff.nix

```diff
@@ -0,0 +1,21 @@
+{ pkgs, lib, config, ... }:
+
+with lib;
+
+let
+  cfg = config.hardware.new-lg4ff;
+in {
+  options.hardware.new-lg4ff = {
+    enable = mkEnableOption "new-lg4ff";
```

Comment:
I am not sure if we should write this by hand and add a more lengthy description. I wouldn't know what lg4ff is tbh.

Reply:
I agree. I've made the required changes.
It should now just work when adding the package to `environment.systemPackages`.
My apologies for the many revisions. I'm still figuring everything out and pretty much use existing packages and nix-template as a reference.
This package is mainly used by sim-racers with Logitech racing wheels that don't have a fully-fledged kernel driver. This will enable force feedback on these wheels and add extra functionalities.

Reply:
If its just added to systemPackages then I don't think we need the module.

Reply:
Well, the module is needed for it to "just work" as a package without any options. Otherwise, user will have to set the boot options manually as I mentioned in the initial commit. I'm pleased with the current implementation but if you insist on the removal, I don't mind changing it again.

Reply:
> It should now just work when adding the package to `environment.systemPackages`.

My comment was based on this. We don't really need a module for that.

Reply:
I see, I guess it was just a misunderstanding. So I understand why it would be best practice to not activate unconditionally. I implemented it this way purely for user-friendliness so nothing needed to be enabled.
I guess it's indeed then better to just add a proper option with a correct description as you initially stated.
Without the module and adding it to systemPackages, it will build but of course not be an active kernel module, so users will still have to figure out what the name is (since it's not the same as the package). This is probably not ideal.
Hopefully, my latest commit is satisfactory.